draft-ietf-wimse-s2s-protocol.md

```diff
@@ -43,6 +43,10 @@ author:
     fullname: "Yaron Sheffer"
     organization: Intuit
     email: "yaronf.ietf@gmail.com"
+ -
+    fullname: "Ken McCracken"
+    organization: Google
+    email: "kenmccracken@google.com"
```

Comment:
IETF Process note - documents should only have at most 5 authors. Other contributors can be listed in the acknowledgements section. It should be fine to add more for now but this will need to be trimmed back prior to ultimate publication.

Reply:
Moved my name to Contributors list at the bottom.